### PR TITLE
Add SQL ShortTermRetention, VulnerabilityAssessment, and Elastic jobs

### PR DESCRIPTION
**Pull request by jaredmoo**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by jaredmoo** [comment]
Swagger PR: https://github.com/Azure/azure-rest-api-specs/pull/2860

I merged from SqlMgmt-CloudLifter branch, updated tests, then regenerated Swagger from commit f943cdccabceca5467712cf1788cbeb52cbfb25b which is the newest commit that does not contain VA (see #4327) or jobs (see #4340)

**Comment by jaredmoo** [comment]
Blocking issues:
- @dealaus - need *Backup*ShortTermRetentionPolicy name change merged into Swagger
- @yaakoviyun - need Execute -> InitiateScan name change and required params merged into Swagger. (Please make a PR separate from your MI change so it can be merged into Swagger immediately)

File: src/SDKs/_metadata/sql_resource-manager.txt [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
-GitHub user: jugeorge
+GitHub user: Azure
 Branch:      master
-Commit:      d08c7f54a125819caaa8c5f553206d1adbae60f9
+Commit:      c6fea13a449c7e60d515759d687f18c9079c8522
 
 2) AutoRest information
 Requested version: latest
-Bootstrapper version:    C:\Users\jugeorge\AppData\Roaming\npm `-- autorest@2.0.4262  
-Latest installed version:
```
Comment (dsgouda) [inline src/SDKs/_metadata/sql_resource-manager.txt]: Please generate the code using [this](https://github.com/jaredmoo/azure-sdk-for-net/blob/8dd820d23bf7b9f5bd35e2a576681b7559af35f8/src/SDKs/SqlManagement/Management.Sql/generate.ps1) script instead

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks great apart from the minor comment

**Comment by jaredmoo** [comment]
Now I am just waiting for https://github.com/Azure/azure-rest-api-specs/pull/3125 to be merged so I can do the final regen.

**Comment by jaredmoo** [comment]
Ready to merge

File: src/SDKs/SqlManagement/Sql.Tests/Sql.Tests.csproj [hunk 2]
```diff
@@ -23,7 +23,7 @@
     <PackageReference Include="WindowsAzure.Storage" Version="8.1.4" />
   </ItemGroup>
   <ItemGroup>
-    <!--<PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.15.0-preview" />-->
```
Comment (dsgouda) [inline src/SDKs/SqlManagement/Sql.Tests/Sql.Tests.csproj]: You don't really need this line

**Review by dsgouda — APPROVED** [review]
LGTM, thanks for addressing all the feedback

**Pull request merged**